test/jdk/sun/security/pkcs11/Provider/MultipleLogins.sh

```diff
@@ -126,10 +126,21 @@ ${TESTJAVA}${FS}bin${FS}java ${TESTVMOPTS} \
         -Dtest.src=${TESTSRC} \
         -Dtest.classes=${TESTCLASSES} \
         -Djava.security.debug=${DEBUG} \
-        MultipleLogins
+        MultipleLogins || exit 10
 
-# save error status
-status=$?
+# run test with security manager
+${TESTJAVA}${FS}bin${FS}java ${TESTVMOPTS} \
+        -classpath ${TESTCLASSPATH} \
+        --add-modules jdk.crypto.cryptoki \
+        --add-exports jdk.crypto.cryptoki/sun.security.pkcs11=ALL-UNNAMED \
+        -DCUSTOM_DB_DIR=${TESTCLASSES} \
+        -DCUSTOM_P11_CONFIG=${TESTSRC}${FS}MultipleLogins-nss.txt \
+        -DNO_DEFAULT=true \
+        -DNO_DEIMOS=true \
+        -Dtest.src=${TESTSRC} \
+        -Dtest.classes=${TESTCLASSES} \
+        -Djava.security.debug=${DEBUG} \
+        MultipleLogins ${TESTSRC}${FS}MultipleLogins.policy || exit 11
```

Comment:
There is no MultipleLogins.policy file. The test just uses the internal SimplePolicy object. Maybe just use a string like "useSimplePolicy".